cmd/zetaclientd/start_utils.go

```diff
@@ -53,41 +52,34 @@ func validatePeer(seedPeer string) error {
 	return nil
 }
 
-// maskCfg sensitive fields are masked, currently only the EVM endpoints and bitcoin credentials,
+// maskCfg sensitive fields are masked, currently only the endpoints and bitcoin credentials,
 //
 //	other fields can be added.
 func maskCfg(cfg config.Config) string {
+	// Make a copy of the config
```

Comment:
Can we use a package for this? https://github.com/showa-93/go-mask 

```go
type FooBar struct {
			ABC string   `mask:"filled"` // will be masked
}
```

Reply:
looks sweet. Let's use it.

Reply:
https://github.com/zeta-chain/node/pull/2870/commits/1b23e969da315ab2e8738d32b5fd0e2068f99358